x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -44,9 +44,18 @@ interface HomePageProps {
 }
 
 const HomePageComponent: React.FC<HomePageProps> = ({ children }) => {
-  const { application } = useKibana().services;
+  const { application, overlays } = useKibana().services;
   const subPluginId = useRef<string>('');
   const { ref, height = 0 } = useThrottledResizeObserver(300);
+  const banners$ = overlays.banners.get$();
+  const [headerFixed, setHeaderFixed] = useState<boolean>(true);
+  const mainPaddingTop = useMemo(() => (headerFixed ? height : 0), [headerFixed, height]);
```

Comment:
Is there a need for `useMemo` here?

Reply:
I believe so, we don't want that prop being updated without the reason